js/popup.js

```diff
@@ -0,0 +1,73 @@
+const cardData = [{
+  title: '',
+  description: '',
+  skills: [''],
+},
+{
+  title: 'Profesional Art Printing Data',
+  description: "A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.",
+  skills: ['HTML', 'Bootstrap', 'Ruby'],
+},
+{
+  title: 'Profesional Art Printing Data',
+  description: "A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.",
+  skills: ['HTML', 'Bootstrap', 'Ruby'],
+},
+{
+  title: 'Profesional Art Printing Data',
+  description: "A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.",
+  skills: ['HTML', 'Bootstrap', 'Ruby'],
+},
+{
+  title: 'Profesional Art Printing Data',
+  description: "A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.",
+  skills: ['HTML', 'Bootstrap', 'Ruby'],
+},
+{
+  title: 'Profesional Art Printing Data',
+  description: "A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.",
+  skills: ['HTML', 'Bootstrap', 'Ruby'],
+},
+];
+
+const cardContainer = document.getElementById('Profile');
+
+function createCard(obj) {
+  const cardDiv = document.createElement('div');
+  cardDiv.className = 'card-hero';
+
+  const article = document.createElement('article');
+  const h2 = document.createElement('h2');
+  h2.textContent = obj.title;
+  article.appendChild(h2);
+
+  const p = document.createElement('p');
+  p.textContent = obj.description;
+  article.appendChild(p);
+
+  const ul = document.createElement('ul');
+  ul.className = 'card-Skills-links';
+
+  const { skills } = obj;
+  skills.forEach((skill) => {
+    const li = document.createElement('li');
+    li.className = `b${skills.indexOf(skill) + 5}`;
+    li.textContent = skill;
+    ul.appendChild(li);
+  });
+
+  article.appendChild(ul);
+
+  const button = document.createElement('button');
+  button.className = 'card-projectbtn';
+  button.textContent = 'See project';
+  button.id = 'showDetailsBtn';
+  article.appendChild(button);
+  cardDiv.appendChild(article);
+  return cardDiv;
+}
```

Comment:
- [ ] The popup in mobile screen has some design flaws. kindly consider fixing them because our design needs to be identical to the figma design:
   -  The top and bottom margin space is missing in your pop-up
   -  The language names should be in boxes and the bullet point should be removed as in the figma deisgn.
   -  The links buttons should be horizontally aligned and the bullet points should be removed.